README.md

```diff
@@ -0,0 +1,144 @@
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📖 [Bookstore]
+
+
+**[Bookstore]**  is an application that allows a user to:
+
+    Display a list of books
+    Add a book.
+    Remove a selected book.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>FrontEnd</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://jestjs.io/docs/tutorial-react">jest</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features 
+
+A react logo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+- [Live Demo Link](coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started 
+
+To get a local copy up and running follow these simple example steps.
+## Prerequisites
+
+  1. You need to have Code Editor Installed
+  2. Make sure Nodejs and NPM is installed in your system for linters and libraries
+
+## Setup
+
+1. Clone the repo in your local machine:
+
+```bash
+git clone git clone https://github.com/khanetmoi/math-magician-app
+```
+2. Open terminal and cd into project and install the dependencies
+```bash
+cd Math-Magician
+npm install
+```
+
+3. run the command to open your index.html automatically in your brower:
+```bash
+npm run build
+npm start
+```
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Inayat-khan**
+
+- GitHub: [@githubhandle](https://github.com/Khanetmoi)
+- Twitter: [@twitterhandle](https://twitter.com/InayatVan)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/khan-bitsindou-b37178228/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features 
+
+  * Use Redux in React components.
+  * connect to API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing 
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Khanetmoi/math-magician-app/issues).
```

Comment:
- [ ] You did a great job working on your project documentation! 💯 However, please consider fixing this link as it is pointing to a different project! 🤓 🔧